balena.yml

```diff
@@ -0,0 +1,30 @@
+
+name: meetbot
+type: sw.application
+description: >-
+  Meetbot is a highly configurable Google meet bot that reduces friction in meetings.
```

Comment:
Why do we always need to speak in extremes 😆 
I think it's more like mid-level configurable 😝

Reply:
I need help over here and README to suggest saner descriptions. This is first draft